modules/substrate/src/lib.rs

```diff
@@ -241,8 +245,14 @@ decl_module! {
 		pub fn set_owner(origin, new_owner: Option<T::AccountId>) {
 			ensure_owner_or_root::<T>(origin)?;
 			match new_owner {
-				Some(new_owner) => ModuleOwner::<T>::put(new_owner),
-				None => ModuleOwner::<T>::kill(),
+				Some(new_owner) => {
+					ModuleOwner::<T>::put(&new_owner);
+					frame_support::debug::info!(target: "sub-bridge", "Setting pallet Owner to: {:?}", new_owner);
```

Comment:
Do we really need the `target`? It will default to module name afair, which is probably good enough.

Reply:
I didn't realize they defaulted to anything, I thought you always had to have one